subsys/bluetooth/mesh/transport.c

```diff
@@ -764,176 +750,46 @@ static void seg_rx_assemble(struct seg_rx *rx, struct net_buf_simple *buf,
 	}
 }
 
-static int remote_devkey_decrypt(struct bt_mesh_net_rx *rx, uint32_t seq, uint8_t *ad,
-				 uint8_t aszmic, struct net_buf_simple *buf,
-				 struct net_buf_simple *sdu)
-{
-	struct bt_mesh_cdb_node *node;
-	int err;
-
-	if (!IS_ENABLED(CONFIG_BT_MESH_PROVISIONER)) {
-		return -ENOENT;
-	}
-
```

Comment:
Mesh Spec `3.4.3 Address validity` Device Key can only used with Unicast address, but i'm not see any check in other place

Reply:
This will be pential bug to cfg_srv,  because model subscribe to config_model not be checked .